src/monaco/monaco.ts

```
@@ -0,0 +1,24 @@
+import * as monaco from "monaco-editor";
+
+// 오버라이드: 워커를 직접 연결
+self.MonacoEnvironment = {
+  getWorker: function (_: unknown, label: string) {
+    switch (label) {
+      case "json":
+        return new (require("worker-loader!monaco-editor/esm/vs/language/json/json.worker.js"))();
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)